src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  &-star {
+    display: flex;
+    background: url(../images/star.svg) center no-repeat;
+    width: 16px;
+    height: 16px;
+    padding: 0;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.stars .stars-star:nth-child(-n + 4) {
+  background: url(../images/star-active.svg) center no-repeat;
+}
```

Comment:
And what if we'll set 3 here? Your stars should have functionality from previous task (if we'll change number in this modifier, it should make needed amount of stars active)